experimental/dds/tree2/src/test/feature-libraries/default-field-kinds/optionalChangeRebaser.spec.ts

```diff
@@ -125,6 +138,32 @@ function rebaseTagged(
 	return currChange;
 }
 
+function rebaseComposed(
+	metadata: RevisionMetadataSource,
+	change: TaggedChange<OptionalChangeset>,
+	...baseChanges: TaggedChange<OptionalChangeset>[]
+): TaggedChange<OptionalChangeset> {
+	baseChanges.forEach((base) => deepFreeze(base));
+	deepFreeze(change);
+
+	const composed = compose(baseChanges);
+	const moveEffects = failCrossFieldManager;
+	const idAllocator = idAllocatorFromMaxId(getMaxId(composed));
+	return tagChange(
+		optionalChangeRebaser.rebase(
+			change.change,
+			// TODO: Is anon change good enough here?
```

Comment:
It seems like it is empirically, but that was a little surprising to me.

Reply:
The top-level revision tag for a composition is indeed expected to be undefined.
The fact that rebase is happy with it is more suprising since we haven't needed that in the past.